Websites/perf.webkit.org/tools/js/buildbot-triggerable.js

```diff
@@ -286,28 +295,30 @@ class BuildbotTriggerable {
             return null;
 
         const isFirstRequest = nextRequest == requestsInGroup[0] || !nextRequest.order();
-        if (!isFirstRequest) {
+        if (!isFirstRequest && nextRequest.isTest()) {
             if (syncer)
-                return this._scheduleRequestWithLog(syncer, nextRequest, requestsInGroup, workerName);
+                return this._scheduleRequestWithLog(syncer, nextRequest, requestsInGroup, workerName, false);
             this._logger.error(`Could not identify the syncer for ${nextRequest.id()}.`);
         }
 
-        // Pick a new syncer for the first test.
-        for (const syncer of this._syncers) {
-            // Only schedule A/B tests to queues whose last job was successful.
-            if (syncer.isTester() && !nextRequest.order() && !syncer.lastCompletedBuildSuccessful())
-                continue;
+        for (const useIdleWorker of [true, false]) {
```

Comment:
This code is rather cryptic. Why are we trying both values?
Don't we need to do this only for "build" build requests?

Reply:
For test type request, this is also a progression, consider following case:

builder 1: 1 running, no pending
builder 2: no running, no pending

Before this change, request may be scheduled on builder 1 if builders are sorted by name, and we introduce extra waiting time.
With this change, builder 2 will be picked and request can be picked up immediately.

Reply:
You mean 1 running from another test group? We clearly don't want to schedule a test request on builder 2 if the one running in builder 1 is of the same test group.

Reply:
Yes, I meant 1 running from another group.